src/usr/local/www/classes/Form/Group.class.php

```diff
@@ -148,7 +148,8 @@ public function __toString()
 		$label = new Form_Element('label', false, ['for' => $target]);
 		$label->addClass('col-sm-'.Form::LABEL_WIDTH, 'control-label');
 
-		$title = htmlspecialchars(gettext($this->_title));
+		if (!empty(trim($this->_title)) || is_numeric($this->_title))
+			$title = htmlspecialchars(gettext($this->_title));
```

Comment:
According developer guide it should be added. @phil-davis spent a long time in the past adding it all around, with all other style fixes. Unfortunately classes code was merged and diverge in some rules, we will need to fix it all in the future, but the idea is to avoid introducing new cases to be fixed later